webpack/components/extensions/HostDetails/Cards/ContentViewDetailsCard/ContentViewDetailsCard.js

```diff
@@ -0,0 +1,202 @@
+import React, { useState } from 'react';
```

Comment:
This file shows as added but really it's just moved (and added the modal to it).